addons/core/addon/helpers/format-day-year.js

```diff
@@ -24,12 +24,14 @@ export default class extends Helper {
    */
 
   compute([days]) {
-    const remainder = days % 365;
-    if (remainder === 0) {
-      const numberOfYears = days / 365;
-      return this.intl.t('titles.year', { numberOfYears });
-    } else {
-      return this.intl.t('titles.days', { days });
+    if (days > 0) {
```

Comment:
None blocking comment, just checking: I did not see test for this helper within this PR, but [I see it here](https://github.com/hashicorp/boundary-ui/commit/d50bc81c5363f6dfdf4cbf18b0f69892936edc2b#diff-6d1c8a8826f8f8018d7586756dc6bfad32bdfc69263c6d91c138738abfc98a73)

Reply:
not sure I follow, the helper has been merged and is already in main. I added an additional check to the helper

Reply:
Resolving this to get this PR merged. Will address it later if it's still a concern